bin/process_snaps.py

```diff
@@ -92,10 +85,6 @@
     "nvidia-core22": {
         "beta": {"recipe": "nvidia-core22-beta"},
     },
-    "mir-kiosk-x11": {
-        "edge": {"ppa": "dev", "recipe": "mir-kiosk-x11-edge", "release": "focal"},
-        "beta": {"ppa": "rc", "recipe": "mir-kiosk-x11-beta", "release": "focal"},
-    },
```

Comment:
We want to keep updating mir-kiosk-x11. We use it in, for example:

https://mir-server.io/docs/packaging-an-x11-application-as-an-iot-gui

Reply:
That won't change existing snaps (which we don't control) that are based on it.